values.yaml

```diff
@@ -820,25 +820,23 @@ redis:
   podAnnotations: {}
 
 exporter:
-    replicas: 1
+  replicas: 1
   # resources:
-  #  requests:
-  #    memory: 256Mi
-  #    cpu: 100m
-    podAnnotations: {}
-    serviceAccountName: ""
-    # mount the service account token
-    automountServiceAccountToken: false
-    image:
-      repository: goharbor/harbor-exporter
-      tag: dev
-    nodeSelector: {}
-    tolerations: []
-    affinity: {}
-    cacheDuration: 23
-    cacheCleanInterval: 14400
-    ## The priority class to run the pod as
-    priorityClassName:
+  #   requests:
+  #     memory: 256Mi
+  #     cpu: 100m
+  podAnnotations: {}
+  serviceAccountName: ""
+  image:
```

Comment:
could you please justify why this PR want to remove these two lines:
```
    # mount the service account token
    automountServiceAccountToken: false
```

Reply:
Apologies if i droped this. I will fix it again, when i can